source/opt/basic_block.h

```diff
@@ -90,6 +101,33 @@ inline void BasicBlock::ForEachInst(
         ->ForEachInst(f, run_on_debug_line_insts);
 }
 
+inline void BasicBlock::ForEachPhiInst(
+    const std::function<void(Instruction*)>& f, bool run_on_debug_line_insts) {
+  for (auto& inst : insts_) {
+    if (inst->opcode() != SpvOpPhi) break;
```

Comment:
No change (yet). My reading of the code is that OpLine (and OpNoLines) preceding an instruction are stored in the dbg_line_insts_ member of the Instruction class, so we shouldn't see them in the insts_ member of a BasicBlock.